### PR TITLE
Bugfix/settings routes access

### LINKED ISSUE #343: Anonymous user can access some /settings/ routes
## Issues
Anonymous user can access next routes:

```
/settings/proxies
/settings/profile
/settings/account
```

## Possible Solution
Check on a route level who can access these routes.

### PR DESCRIPTION
**Pull request by nnn**
Closes #343

### CODE REVIEW

**Comment by nnn** [comment]
@Digipalvelutehdas/apika-tiimi please review as soon as possible.

File: setup/client/lib/router.js [hunk 1]
```diff
@@ -1,21 +1,5 @@
 import { FlowRouter } from 'meteor/kadira:flow-router';
```
Comment (mauriciovieira) [inline setup/client/lib/router.js]: Please remove `setup/client/lib/router.js` from `.eslintignore` and lint.

File: users/client/lib/router.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Roles } from 'meteor/alanning:roles';
 import { sAlert } from 'meteor/juliancwirko:s-alert';
```
Comment (mauriciovieira) [inline users/client/lib/router.js]: please remove `users/client/lib/router.js` from `.eslintignore` and lint.

**Comment by mauriciovieira** [comment]
Also try `node_modules/bin/eslint --fix .` for autofixes

**Comment by nnn** [comment]
@mauriciovieira done.

**Comment by brylie** [comment]
@mauriciovieira will you finish this PR? E.g. just review it until it seems ready, as you are already doing.

**Comment by brylie** [comment]
@nnn please resolve conflicts.

**Comment by mauriciovieira** [comment]
LGTM

**Pull request merged**